libs/src/oracle2/types.ts

```diff
@@ -1,144 +1,20 @@
+import type {
+  Allowances,
+  Assertions,
+  Balances,
+  Requests,
+  Tokens,
+  Transactions,
+} from "@shared/types";
+
 export {
-  Provider,
-  JsonRpcSigner,
+  FallbackProvider,
   JsonRpcProvider,
+  JsonRpcSigner,
+  Provider,
   Web3Provider,
-  FallbackProvider,
 } from "@ethersproject/providers";
 
-export enum RequestState {
```

Comment:
these types are moved to @shared

Reply:
Very much agree. This is something I often struggle with. I'm going to message you later to get some more advice about doing this in the future